include/deal.II/base/tensor.h

```diff
@@ -1008,7 +1006,7 @@ DEAL_II_CONSTEXPR inline DEAL_II_ALWAYS_INLINE
 
 
 template <int dim, typename Number>
-DEAL_II_CUDA_HOST_DEV inline DEAL_II_ALWAYS_INLINE
+DEAL_II_CONSTEXPR DEAL_II_CUDA_HOST_DEV inline
```

Comment:
Is the removal of the `always_inline` attribute going to upset anything here?

Reply:
I don't think it's critical here, except that the function is cheap enough that it would not hurt.

Reply:
That's a rebase artifact. `DEAL_II_CONSTEXPR` should be here.